portal/src/texts/profile/colors.md

```diff
@@ -1,3 +1,5 @@
 ## Farger
 
-Fremtind bruker få farger. Hovedfargene er fra primærpaletten, med fokus på hvit og svart. Vi bruker støttefargene/funksjonsfargene der det kan gi bedre forståelse for hva som skjer eller hva brukeren skal gjøre i løsningen. I og med at vi har en nøktern fargepalett, skaper vi liv og fokus med andre virkemidler, for eksempel med animasjon.
+Fremtinds hovedfarger er varme gråtoner som gir assosiasjoner til fjellet, og til nordisk minimalisme. Støttefargene henter inspirasjon fra den norske naturen, og der hovedfargene representerer fjell og vidde, viser støttefargene strandkanten og åssidene. I tillegg til hoved- og støttefargene finner vi noen funksjonsfarger som utfyller for bruk i digitale flater.
+
+Jøkul benytter seg i størst grad av hovedfargene—særlig sort og hvitt—i komponentene. Støtte- og funksjonsfargene benyttes der man vil formidle viktig informasjon, eller der det letter forståelsen.
```

Comment:
Forslag: Vi bruker helst hovedfargene mest mulig, særlig svart og hvitt, når vi lager komponenter. Støtte- og funksjonsfargene bruker vi når vi vil formidle viktig informasjon, eller der det gjør det lettere for brukerne å forstå løsningen.